### DISCUSSION TITLE
I can't run the code :/

### DISCUSSION THREAD

**Discussion by mehmet0150** [halaxa/json-machine#45] [Q&A] [answered]
Hello, I'm trying to run a page through hosting. I unzipped the files I downloaded from GitHub and uploaded them to the home directory. Here is the summary of my codes:

```php
<?php

use JsonMachine\JsonMachine;

$URL = 'https://www.binance.com/api/v3/depth?limit=5000&symbol=ETHBTC';
$JSON = file_get_contents($URL);
$obj = JsonMachine::fromFile($JSON);
	
echo "<PRE>";
var_dump($obj);

?>
```

I am a little newbie in PHP. I guess I can't call the code correctly. Can you help with this?

**Comment by meduzen** [comment, marked as answer]
I think you should use `fromString` instead of `fromFile`.

**Reply by mehmet0150** [reply]
Thanks for your answer. I'm getting the "This page is not working HTTP 500" error.

**Reply by halaxa** [reply]
Use some debugging tool such as https://tracy.nette.org/en/ or https://symfony.com/doc/current/profiler.html so the error 500 is more verbose and you know what's going on. Guessing a 500 error is impossible.

**Reply by mehmet0150** [reply]
The problem was with Composer. I did the necessary installations manually. I think it is working right now. Thank you again.
